src/sidebar/widget-controller.js

```diff
@@ -33,7 +33,7 @@ function groupIDFromSelection(selection, results) {
 // @ngInject
 module.exports = function WidgetController(
   $scope, annotationUI, annotationMapper, drafts, features, frameSync, groups,
```

Comment:
For functions that have parameters injected by Angular DI we have a convention that the params should be alphabetically ordered with Angular `$service`s first. This makes it easier to scan the list and see whether a particular service is already injected.